shared-articles/what does dcdd do.md

```diff
@@ -0,0 +1,90 @@
+
+Wallets running in fully validating mode allow for maximum personal privacy and security. On the other hand, SPV mode is great for mobile wallets or when a device has limited resources. 
+
+**Lightning Network Wallets** are run as fully validating nodes and for maximum accessibility can have an open inbound connection. LN operators are also incentivised, to run a full node that supports the network, with the fees from conducting transactions over the Lightning Network.
+
+**Voting Service Providers (VSP)** are run as fully validating nodes and for maximum accessibility can have an open inbound connection. VSP operators are also incentivised with the fees for conducting ticket transactions. For the best performance, these nodes should run constantly 24/7/365 and have +4 instances to increase reliability.
+
+**DCRDEX Operators** need access to trusted full nodes for each of the assets supported. While operation via a surrogate blockchain data service such as a block explorer is potentially feasible, as it is with all Decred services. It would entail significant security risks and threat of funds being stolen.  
+
+## Conclusion
+Running a fully validating node is the best way of understanding how the network is performing. When you participate in the decentralisation of the network, you understand what it takes to run it and the levels to which you are contributing. 
+
+It occurs to me, from my research here, that decentralisation is a collective endeavour, and making the system and your investment more secure means your participation is necessary. In centralised products, you can rely on someone else to run the system. In fully decentralised products, you are the system and the responsibility sits squarely at your feet.
+
+In the past, I’ve been criticised for liking the fact that Decred’s blockchain size is so small. My rebuttal has always been, “Try running your favourite blockchain, and then you’ll understand!”. I relish in the knowledge that every time I turn on Decrediton (in fully validation mode) I’m looking at the full blockchain and have the latest information at my fingertips. 
+
+From a personal perspective, I’m secure! The next step for me is to add security to the network by running a top-tier node with an open inbound connection running 24/7/365.
+
+## Technical specifications for running dcrd
+* 16 GB disk space (as of April 2022, increases over time, 2 GB/yr)
+* 2 GB memory (RAM)
+* 150 MB/day download, 1.5 GB/day upload
+* Plus one-time initial download of the entire blockchain
+* Windows 11, OpenBSD and FreeBSD are also supported
```

Comment:
Let's return Linux and macOS to this list - these are the two major OSes besides Windows.

Also I suspect Windows version may be lower but better check it elsewhere.